### DISCUSSION TITLE
Accessing Query parameters in response function in Javascript

### DISCUSSION THREAD

**Discussion by Aditya20kul** [Kong/kong#8184] [Help] [answered]
Hi All,
I want to access the query parameters passed inside the response function in javascript, is there a way to do it ?
Thanks in advance !

**Comment by kikito** [comment, marked as answer]
Hello, there's a javascript sdk for writing plugins, you can find more information about it here:

https://docs.konghq.com/gateway/2.7.x/reference/external-plugins/#developing-javascript-plugins

https://github.com/Kong/kong-js-pdk

**Reply by fffonion** [reply]
@Aditya20kul Could you elaborate bit more about your use case? Are you tring to transform a JSONP response?
